src/server/yamlServer.ts

```diff
@@ -114,11 +122,16 @@ function getSuggestions(rowIndex: number, fileUrl: URL) {
             });
         }
     }
+    if(completionItems.length > 0) {
+        autoCompleteTelemetryData.addProperty(TelemetryConstants.AUTO_COMPLETE_RESULT, TelemetryConstants.SUCCESS);
+        autoCompleteTelemetryData.addMeasurement(TelemetryConstants.COUNT_OF_AUTOCOMPLETE_RESULTS, completionItems.length);
+        sendTelemetryEvent(connection, autoCompleteTelemetryData); // we send telemetry data only in case of success otherwise the logs will be bloated with unnecessary data
```

Comment:
ahh, that's a good comment. I would put it on its own line at the start of the if-block (or on the line just before it) so it's clearer at the point of decision.